saged/models.py

```diff
@@ -206,9 +222,12 @@ def __init__(self,
         Arguments
         ---------
         seed: The random seed to use in training
+        lr: The learning rate to be used in training
```

Comment:
Shouldn't this be something related to the `l2_penalty`?

Also it might be good to mention that this is actually the _inverse_ of the standard L2 penalty (see the documentation for `C`[here](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html)). This shouldn't affect your results much if you're using an automatic optimization framework like optuna, just good to keep in mind.

Reply:
Good catch, I deleted the wrong comment